aws-rds-optiongroup/src/main/java/software/amazon/rds/optiongroup/CallbackContext.java

```diff
@@ -7,4 +7,5 @@
 @lombok.ToString
 @lombok.EqualsAndHashCode(callSuper = true)
 public class CallbackContext extends StdCallbackContext {
+    private String optionGroupArn;
```

Comment:
Could you elaborate why this needed. Why you are using it by calling `context.getOptionGroupArn()`?